### DISCUSSION TITLE
Android version

### DISCUSSION THREAD

**Discussion by xizhengchao** [c-smile/sciter-sdk#197] [General]
Excuse me,When will the Android version be released?

**Comment by c-smile** [comment]
Sorry, I am busy with other tasks at the moment. 

Working Sciter's Android version already in our SVN here. The only thing that I haven't decided yet is its final packaging,
Most probably I will package it as a [Quark binary](https://quark.sciter.com/).

**Comment by ivan-ushakov** [comment]
Any updates? What about iOS version?
